package/resources/data/uom_seed.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="uom_category_bahmni" model="uom.category">
+        <field name="name">Bahmni</field>
+    </record>
+    <record id="uom_uom_1" model="uom.uom">
+        <field name="name">Unit(s)</field>
```

Comment:
This is a redundant Unit which is also provided by Odoo default. Creating with the same name in a different category will cause confusion while the Syncable units mapping is filtered from the API call.